bower.json

```diff
@@ -12,17 +12,17 @@
     "angular-animate": "1.5.11",
     "angular-touch": "1.5.11",
     "angular-route": "1.5.11",
-    "angular-bootstrap": "0.14.3",
-    "angular-patternfly": "3.24.0",
+    "angular-bootstrap": "2.2.x",
+    "angular-patternfly": "4.1.1",
     "angular-gettext": "2.3.9",
     "uri.js": "1.18.0",
     "moment": "2.14.2",
     "moment-timezone": "0.5.3",
-    "patternfly": "3.24.0",
+    "patternfly": "3.25.1",
     "hawtio-core": "2.0.37",
     "hawtio-extension-service": "2.0.2",
-    "jquery": "2.1.4",
-    "lodash": "3.10.1",
+    "jquery": "3.2.1",
+    "lodash": "4.x",
```

Comment:
Yeah. In the past we've locked to versions for repeatable builds, so let's be consistent here.

This used to be mandatory when we were in the openshift/origin repo to avoid breaking the merge queue. We could opt to be more flexible now that we're in our own repo, but for now let's lock to 4.17.4